gitlab/v4/objects/iterations.py

```diff
@@ -16,11 +17,37 @@ class GroupIterationManager(ListMixin, RESTManager):
     _path = "/groups/{group_id}/iterations"
     _obj_cls = GroupIteration
     _from_parent_attrs = {"group_id": "id"}
-    _list_filters = ("state", "search", "include_ancestors")
+    # When using the API, the "in" keyword collides with python's "in" keyword
+    # raising a SyntaxError.
+    # For this reason, we have to use the query_parameters argument:
+    # group.iterations.list(query_parameters={"in": "title"})
+    _list_filters = (
+        "include_ancestors",
+        "include_descendants",
+        "in",
+        "search",
+        "state",
+        "updated_after",
+        "updated_before",
+    )
+    _types = {"scopes": types.ArrayAttribute}
```

Comment:
Sorry, I did a copy/paste derp while suggesting this. I'll apply this now and squash the PR.

```suggestion
    _types = {"in": types.ArrayAttribute}
```